CMake/resolve_dependency_modules/qatzstd.cmake

```
@@ -0,0 +1,80 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+include_guard(GLOBAL)
+
+include(Findzstd)
```

```suggestion
if(NOT TARGET zstd::zstd)
  find_package(zstd REQUIRED)
endif()
```
`find_package` calls are cached and if the target exists we already called that file and the variables are set. This way we prevent any potential problems (however unlikely) of different versions of zstd being found and used across velox.